dpub/core.py

```diff
@@ -7,18 +7,19 @@
 
 from dpub import cli, drive, output, parser, pipe
 from dpub.ref import (extend_cell_location_to_range, join_location, next_cell,
-                      split_location)
+                      split_location, split_cell, join_cell, cell_for_new_test)
 
 
 class DPubError(Exception):
     pass
 
 
 class Test:
-    def __init__(self, id, first_message_range):
+    def __init__(self, id, first_message_range, update):
         self.id = id
         self.first_message_range = first_message_range
         self.items = []
+        self.update = update
```

Comment:
As this is an artificial flag, maybe it would be help using an enumeration instead of a string.
You can define enums in a specific class for that:

```python
from enum import Enum

@unique
class Update(Enum):
    NEW = 0,
    YES = 1,
    NO = 2,
```
however if you follow my explanation below, maybe you don't need such flag

Reply:
Ok, good to know about that! So we'll work to avoid the update field.